lib/spot-checker.js

```diff
@@ -97,11 +96,9 @@ export default class SpotChecker {
    * @param {string} args.cid
    * @param {object} args.stats
    * @param {'all' | 'block'} args.dagScope
-   * @param {number} [args.maxByteLength=0] - Maximum bytes to download before stopping retrieval (applies to full retrievals only)
+   * @param {number} [args.maxByteLength]
```

Comment:
Should we keep the param documentation?

```suggestion
   * @param {number} [args.maxByteLength] Maximum bytes to download before stopping retrieval (optional)
```